components/movementsensor/server.go

```diff
@@ -172,10 +172,24 @@ func (s *serviceServer) GetAccuracy(
 		return nil, err
 	}
 
+	uacc := UnimplementedOptionalAccuracies()
+
 	if accuracy != nil {
+		// check optional accuracies
+		switch {
+		case &accuracy.Hdop == nil:
+			accuracy.Hdop = uacc.Hdop
+		case &accuracy.Vdop == nil:
+			accuracy.Vdop = uacc.Vdop
+		case &accuracy.CompassDegreeError == nil:
+			accuracy.CompassDegreeError = uacc.CompassDegreeError
+		case &accuracy.NmeaFix == nil:
+			accuracy.NmeaFix = uacc.NmeaFix
+		}
```

Comment:
will users know that these values are meant to be ignored?

Reply:
So they're completely nonsense values (hdop, vdop and compassheading are math.NaN, nmeaFix is -1). Users will need to look up what they mean through our docs, because they're not super "intuitive" to the user. But right now I need to figure out how to make the server set the defaults properly to the accuracy response.